contracts/tests/HotfixClaimRevenue.t.sol

```diff
@@ -0,0 +1,334 @@
+
+        // 16_678_635: borrower calls claim revenue
+        _rollAndWarpToBlock(16_678_635); 
+        vm.startPrank(borrower);
+        emit log_string("=> claimRevenue()");
+        uint256 claimed = spigot.claimRevenue(BORROWER_REVENUE_EOA, USDC, bytes(""));
+        spigotOwnerTokens = spigot.getOwnerTokens(USDC);
+        spigotOperatorTokens = spigot.getOperatorTokens(USDC);
+        emit log_named_uint("owner tokens USDC", spigotOwnerTokens);
+        emit log_named_uint("operator tokens USDC", spigotOperatorTokens);
+        uint256 expectedOwnerTokens = (claimed * revenueSplit) / 100;
+        assertEq(spigotOwnerTokens, expectedOwnerTokens);
+        assertEq(spigotOperatorTokens, claimed - expectedOwnerTokens);
+        vm.stopPrank();
+
+        // 16_685_641: arbiter calls claimAndTrade ( 13500000 in)
+        _rollAndWarpToBlock(16_685_641); 
+        vm.startPrank(arbiter);
+        emit log_string("=> claimAndTrade()");
+        line.claimAndTrade(USDC,hex"d9627aa400000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000cdfe60000000000000000000000000000000000000000000000000b9c4f4da1409985600000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000000000000000000000006b175474e89094c44da98b954eedeac495271d0f869584cd000000000000000000000000e9039a6968ed998139e023ed8d41c7fa77b7ff7a0000000000000000000000000000000000000000000000cb3b1dde6d63f65891");
+        spigotOwnerTokens = spigot.getOwnerTokens(USDC);
+        spigotOperatorTokens = spigot.getOperatorTokens(USDC);
+        emit log_named_uint("owner tokens USDC after claimAndTrade", spigotOwnerTokens);
+        emit log_named_uint("operator tokens USDC after claimAndTrade", spigotOperatorTokens);
+        vm.stopPrank();
+
+        // 16_685_974: borrower calls useAndRepay
+        _rollAndWarpToBlock(16_685_974); 
+        (, uint256 principal,uint256 interest,,,,,) = line.credits(line.ids(0));
+        emit log_named_uint("principal", principal);
+        emit log_named_uint("interest", interest);
+        
+        vm.startPrank(borrower);
+        emit log_string("=> useAndRepay()");
+        line.useAndRepay(10 ether);
+        vm.stopPrank();
+
+        spigotOwnerTokens = spigot.getOwnerTokens(USDC);
+        spigotOperatorTokens = spigot.getOperatorTokens(USDC);
+        assertEq(IERC20(USDC).balanceOf(address(spigot)), spigotOperatorTokens);
+        emit log_named_uint("owner tokens USDC after useAndRepay", spigotOwnerTokens);
+        emit log_named_uint("operator tokens USDC after useAndRepay", spigotOperatorTokens);
+        emit log_named_uint("line USDC balance", IERC20(USDC).balanceOf(lineAddress));
+        emit log_named_uint("spigot USDC balance", IERC20(USDC).balanceOf(address(spigot)));
+
+        // full 15 USDC has been used at this point
```

Comment:
we only called claimAndTrade to use 13.5 USDC so when did we use the other 1.5 USDC if it wasnt the operator tokens? If the bug is already present here then shouldnt there be an error in the check above `assertEq(IERC20(USDC).balanceOf(address(spigot)), spigotOperatorTokens);`?

Reply:
If you're referring to the comment, I think I just put this as a marker for myself when I was tracing the sequence of transactions. I'll remove it to avoid confusion.